config/locales/en/conviction.yml

```diff
@@ -30,3 +30,7 @@ en:
           page_title: conviction date
           heading:  When did the conviction end?
           lead_text: Date of conviction
+      conviction_length:
+        edit:
+          page_title: conviction length
+          heading:  Was the length of conviction given in weeks or months?
```

Comment:
```suggestion
          heading: Was the length of conviction given in weeks or months?
```

Reply:
If we are using the fieldset then we don't need heading, otherwise, leave it, but better not to duplicate if one of the two locales is no needed.